web/src/subscriber_api.ts

```diff
@@ -39,14 +40,14 @@ export function add_user_ids_to_stream(
 }
 
 export function remove_user_id_from_stream(
-    user_id: number[],
+    user_id: number,
     sub: StreamSubscription,
-    success: () => void,
+    success: (data: unknown) => void,
     failure: (xhr: JQuery.jqXHR<unknown>) => void,
-): JQuery.jqXHR<unknown> | undefined {
+): void {
     // TODO: use stream_id when backend supports it
     const stream_name = sub.name;
-    return channel.del({
+    void channel.del({
```

Comment:
This looks like more than a types change -- we're actually changing the interface. (Probably fine, but the commit message should be clearer, like `subscriber_api: Don't return unused channel request objects.")

Reply:
Fixed. Thanks.